stackit/internal/services/loadbalancer/loadbalancer_acc_test.go

```diff
@@ -97,6 +101,7 @@ func configResources(targetPort string) string {
 		loadBalancerResource["interval_jitter"],
 		loadBalancerResource["timeout"],
 		loadBalancerResource["unhealthy_threshold"],
+		loadBalancerResource["use_source_ip_address"],
```

Comment:
this must go after `target_display_name`

Reply:
done